cmd/up/exec_unix.go

```diff
@@ -8,14 +8,13 @@ import (
 	"os"
 	"os/exec"
 	"runtime"
-	"strings"
 	"syscall"
 )
 
 func (he *hybridExecutor) GetCommandToExec(ctx context.Context, cmd []string) (*exec.Cmd, error) {
 	var c *exec.Cmd
 	if runtime.GOOS != "windows" {
-		c = exec.Command("bash", "-c", strings.Join(cmd, " "))
+		c = exec.Command(cmd[0], cmd[1:]...)
```

Comment:
I think we should make this change in another PR (the one about Hybrid executors), also because by removing the wrapping, we break the use case where `command: script.sh`

Reply:
We agred to keep this change and resolve the issue targetted by this PR and address other use-cases in a follow up PR